### PR TITLE
Remove >>> wrapping <<< if folding is supported

### PR DESCRIPTION
**Pull request by rhaschke**
If folding is supported, the extra delimiters (>>>> <<<<) are more distracting than helping.
This shifts those delimiters to `none.sh`.

### CODE REVIEW

File: industrial_ci/src/folding/none.sh [hunk 1]
```diff
@@ -17,8 +17,10 @@
 
 function  ici_start_fold() {
     shift 3
+    ici_color_output $ANSI_BLUE ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
```
Comment (ipa-mdl) [inline industrial_ci/src/folding/none.sh:20]: To make shellcheck happy:
```suggestion
    ici_color_output "$ANSI_BLUE" ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
```

File: industrial_ci/src/folding/none.sh [hunk 2]
```diff
@@ -17,8 +17,10 @@
...
 }
 
 function  ici_end_fold() {
     shift 4
+    ici_color_output "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
```
Comment (ipa-mdl) [inline industrial_ci/src/folding/none.sh:25]: color is set beforehand..
```suggestion
    echo "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
```

**Review by ipa-mdl — CHANGES_REQUESTED** [review]
This will take some time to get used to ;)

Looks much cleaner on Github. I will push tests to Gitlab and Travis as well.

**Comment by ipa-mdl** [comment]
[Travis output](https://travis-ci.com/github/ipa320/industrial_ci_mirror/jobs/497246577) looks fine, but [Gitlab](https://gitlab.com/ipa-mdl/industrial_ci/-/jobs/1165245869) does not. It might be better to copy the wrapping to gitlab.sh as well.

**Comment by rhaschke** [comment]
> But [Gitlab](https://gitlab.com/ipa-mdl/industrial_ci/-/jobs/1165245869) does not look nice. It might be better to copy the wrapping to gitlab.sh as well.

I would say it does: The fold markers nicely indicate the start of a new section. I think, it's a matter of taste / habituation.
I pushed another commit to [collapse folds by default](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/42231), so it should look exactly like github / Travis.

**Comment by ipa-mdl** [comment]
> I pushed another commit to collapse folds by default, so it should look exactly like github / Travis.

[Very nice](https://gitlab.com/ipa-mdl/industrial_ci/-/jobs/1165356952)!

File: industrial_ci/src/folding/gitlab.sh [hunk 3]
```diff
@@ -19,7 +19,7 @@ function  ici_start_fold() {
     shift
     local name=$1; shift
     local start=$1; shift
-    ici_ansi_cleared_line "section_start:${start::-9}:$name"
+    ici_ansi_cleared_line "section_start:${start::-9}:$name[collapsed=true]"
```
Comment (ipa-mdl) [inline industrial_ci/src/folding/gitlab.sh:22]:
```suggestion
    ici_ansi_cleared_line "section_start:${start::-9}:${name}[collapsed=true]"
```

**Review by ipa-mdl — APPROVED** [review]
~~Just one minor shellcheck issue.~~
If it is okay for you, I will squash-merge once the tests pass.

**Comment by rhaschke** [comment]
> If it is okay for you, I will squash-merge once the tests pass.

Sure!

**Pull request merged**